### ISSUE TITLE
[Feature Request] Highlight the line in Current Lyrics as it plays.

### ISSUE THREAD

**Issue by maggitt** [cvzi/Youtube-Genius-Lyrics-userscript#61]
Text is scrolling-down properly as a song plays, but the current line is not highlighted, which makes it hard to follow the currently heard lyrics.
Adding some highlight to Current + Next line would be ideal.

**Comment by cvzi** [comment]
The scrolling-down of the lyrics is only an approximation. We just use the song position and estimate where the lyrics probably are.
genius.com doesn't offer time-codes for the lyrics (afaik), so we can't know which line is actually being heard.
